deeppavlov/models/doc_retrieval/logit_ranker.py

```diff
@@ -33,50 +32,69 @@ class LogitRanker(Component):
         squad_model: a loaded squad model
         batch_size: batch size to use with squad model
         sort_noans: whether to downgrade noans tokens in the most possible answers
+        top_n: number of answers to return
 
      Attributes:
         squad_model: a loaded squad model
         batch_size: batch size to use with squad model
+        top_n: number of answers to return
 
     """
 
     def __init__(self, squad_model: Union[Chainer, Component], batch_size: int = 50,
-                 sort_noans: bool = False, **kwargs):
+                 sort_noans: bool = False, top_n: int = 1, **kwargs):
         self.squad_model = squad_model
         self.batch_size = batch_size
         self.sort_noans = sort_noans
+        self.top_n = top_n
+
+    def __call__(self, contexts_batch: List[List[str]], questions_batch: List[List[str]],
+                 tfidf_doc_ids: Optional[List[List[str]]] = None) -> \
+            Union[Tuple[List[str], List[float], List[int], List[str]],
+                  Tuple[List[List[str]], List[List[float]], List[List[int]], List[List[str]]]]:
```

Comment:
fix typing in case if tfidf_doc_ids is None
```suggestion
            Union[
                Tuple[List[str], List[float], List[int], List[str]],
                Tuple[List[List[str]], List[List[float]], List[List[int]], List[List[str]]],
                Tuple[List[str], List[float], List[int]],
                Tuple[List[List[str]], List[List[float]], List[List[int]]]
                ]:
```